hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/record/historicalbalance/HistoricalBalanceService.java

```diff
@@ -45,9 +46,9 @@
 import org.springframework.transaction.support.TransactionTemplate;
 
 @ConditionalOnProperty(
-        prefix = "hedera.mirror.importer.parser.record.historical-balances",
         name = "enabled",
-        matchIfMissing = true)
+        matchIfMissing = true,
+        prefix = "hedera.mirror.importer.parser.record.historical-balances")
```

Comment:
I missed it in my PR...
```suggestion
        prefix = "hedera.mirror.importer.parser.record.historical-balance")
```

Reply:
Fixed